stock_vertical_lift/models/stock_move_line.py

```diff
@@ -1,12 +1,21 @@
 # Copyright 2019 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models
+from odoo import fields, models
 
 
 class StockMoveLine(models.Model):
     _inherit = "stock.move.line"
 
+    vertical_lift_pick_skipped = fields.Boolean(
```

Comment:
Is it going to be implemented for the Put screen? I think `vertical_lift_skipped` is enough, because in any case the move line can't be both put and pick, so the field can be reused if ever we add skip on putaway lines